.travis.yml

```diff
@@ -1,7 +1,7 @@
 language: node_js
 
 node_js:
-  - "8.10.0"
+  - "11.6.0"
```

Comment:
was there a reason to select node 11?
![image](https://user-images.githubusercontent.com/1764195/93603646-97912700-f9c4-11ea-92ad-805889720173.png)

Reply:
https://nodejs.org/en/about/releases/

Reply:
1 of the dev dependancies needs a newer version to build it. I think its mocha not 100% sure

Reply:
I mean why v11 and not 10/12? both are LTS

Reply:
Ah ill change it to 12. 11 was the one my laptop was using by default. so i was sure that it works :P